tools/verdict-back-ends/verdict-bundle-parent/verdict-bundle/src/main/java/com/ge/verdict/bundle/App.java

```diff
@@ -533,58 +540,68 @@ private static void runGsn(
 
                     try {
                         createGsnObj.runGsnArtifactsGenerator(
                                 id,
                                 gsnOutputDir,
                                 soteriaOutputDir,
-                                caseAadlPath,
+                                modelAadlPath,
                                 securityCases,
                                 generateXml,
-                                soteriaOutputLinkPathPrefix);
+                                soteriaOutputLinkPathPrefix,
+                                hostSTEMDir);
                     } catch (IOException | ParserConfigurationException | SAXException e) {
                         // TODO Auto-generated catch block
                         throw new VerdictRunException("Failed to create GSN fragments", e);
                     }
                 } else {
-                    // calling the function to create GSN artefacts
+                    // calling the function to create normal GSN artefacts
                     GSNInterface createGsnObj = new GSNInterface();
 
                     try {
                         createGsnObj.runGsnArtifactsGenerator(
                                 id,
                                 gsnOutputDir,
                                 soteriaOutputDir,
-                                caseAadlPath,
+                                modelAadlPath,
                                 generateXml,
-                                soteriaOutputLinkPathPrefix);
+                                soteriaOutputLinkPathPrefix,
+                                hostSTEMDir);
                     } catch (IOException | ParserConfigurationException | SAXException e) {
                         // TODO Auto-generated catch block
                         throw new VerdictRunException("Failed to create GSN fragments", e);
                     }
                 }
             } else { // if not cyberId
-                // calling the function to create GSN artefacts
+                // calling the function to create normal GSN artefacts
                 GSNInterface createGsnObj = new GSNInterface();
 
                 try {
                     createGsnObj.runGsnArtifactsGenerator(
                             id,
                             gsnOutputDir,
                             soteriaOutputDir,
-                            caseAadlPath,
+                            modelAadlPath,
                             generateXml,
-                            soteriaOutputLinkPathPrefix);
+                            soteriaOutputLinkPathPrefix,
+                            hostSTEMDir);
                 } catch (IOException | ParserConfigurationException | SAXException e) {
                     // TODO Auto-generated catch block
                     throw new VerdictRunException("Failed to create GSN fragments", e);
                 }
             }
         }
 
+        try {
+            Thread.sleep(3000);
```

Comment:
Please call TimeUnit.SECONDS.sleep(3) instead for readability.  <https://stackoverflow.com/questions/9587673/thread-sleep-vs-timeunit-seconds-sleep>